R/assertions.R

```diff
@@ -915,8 +907,8 @@ assert_named_exprs <- function(arg, optional = FALSE) {
 #'
 #' @export
 #'
-#' @keywords assertion
-#' @family assertion
+#' @keywords deprecated
+#' @family deprecated
```

Comment:
Do you mind deleting the examples to follow our deprecation strategy?

![image](https://github.com/pharmaverse/admiraldev/assets/10111024/51c60fb1-022e-45c9-9ee0-a9034573c4c7)